user-guide/settings/https.md

```diff
@@ -6,25 +6,25 @@ publish: true
 position: 10
 ---
 
-## HTTPS Settings
+# HTTPS Settings
 
 After the initial startup, the Fiddler Everywhere application could only capture non-secure traffic (HTTP) while SSL traffic is not captured. To enable capturing and decrypting HTTPS traffic, you will need to explicitly install a root trust certificate via the __HTTPS__ submenu in __Settings__.
 
 ![default https settings](../../images/settings/settings-https.png)
 
-#### Trust Root Certificate
+### Trust Root Certificate
 
 The button installs and trusts the Fiddler root certificate (macOS and Windows only).
 
-#### Capture HTTPS traffic
+### Capture HTTPS traffic
 
-The option defines if Fiddler will capture HTTPS traffic or will skip it. It is inactive by default.  To active it, the root certificate should be trusted first.
+The option defines if Fiddler Everywhere should capture HTTPS traffic or skip it. By default, this option is disabled. To enable the HTTPS traffic capturing, firstly you need to confirm whetherthe root certificate trusted first. If it is trusted you can simply enable the capture option.
```

Comment:
1) Missing white space between **whether** and **the**
2) The sentence "To enable the HTTPS traffic capturing, firstly you need to confirm whether the root certificate trusted first." sounds strange and seems to be missing a word or two. 
- I would avoid using "firstly". It's correct, but it's not used often and people might wonder if such word exists or not.
- I think there should be an "is" before **trusted** and the final "first" should be removed.

Here is my suggestion how it should look:
 _To enable the HTTPS traffic capturing, first you need to confirm whether the root certificate is trusted._